### PR TITLE
Add Impressum Site, amend display of Breadcrumbs. testing: minor edits

### PR DESCRIPTION
**Pull request by Th3da**
Add imprint page, amend display of breadcrumbs, minor edits to warnings during tests, correcting some typos.

Maybe in the future we want to provide the imprint text in another way - for the moment it's hard coded.

Is **e-recht24.de** really a good source for our imprint statement?

### CODE REVIEW

File: src/app/app.module.ts [hunk 1]
```diff
@@ -80,7 +81,8 @@ export function createTranslateLoader(http: HttpClient) {
     HelperModule,
     OpenCaseListSliderModule,
     HomeModule,
-    MyProfileModule
+    MyProfileModule,
+    BreadcrumbModule
```
Comment (SandroGlueck) [inline src/app/app.module.ts:85]: Is that import required? Isn't the `BreadcrumbModule` intended to only be used in the "functional" modules?

File: src/app/core/components/imprint/imprint.component.scss [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+      font-size: 16px;
+      font-weight: 700;
+      color: #8989a2;
+      margin-top: 25px;
+      margin-bottom: 8px;
+    }
+    
+    p {
+      font-size: 16px;
+      color: #8989a2;
```
Comment (SandroGlueck) [inline src/app/core/components/imprint/imprint.component.scss:29]: The color codes #4a4a4a and #8989a2 are bot available from the shared `_colors.scss` stylesheet.
Please import that stylesheet and use `$color__old-paragraph` and `$color__share-section-heading` respectively.

File: src/app/shared/breadcrumb/breadcrumb/breadcrumb.component.html [hunk 3]
```diff
@@ -1,8 +1,9 @@
 <span class="breadcrumb">
   <i class="far fa-home breadcrumb__link" routerLink="/"></i>
+  <i class="fal fa-greater-than"></i>
   <ng-container *ngFor="let breadcrumbLink of links; let last = last">
     <span class="breadcrumb__link" [routerLink]="breadcrumbLink.link">{{ breadcrumbLink.label }}</span>
     <i class="fal fa-greater-than" *ngIf="!last"></i>
   </ng-container>
-  <i class="fal fa-greater-than"></i>
+
```
Comment (SandroGlueck) [inline src/app/shared/breadcrumb/breadcrumb/breadcrumb.component.html:8]: While at it, you could remove this empty line 😄

**Review by SandroGlueck — CHANGES_REQUESTED** [review]
Looking great - only a few minor remarks 😃

File: src/app/main-layout/main-layout.component.spec.ts [hunk 4]
```diff
@@ -16,6 +18,9 @@ describe('MainLayoutComponent', () => {
         RouterTestingModule.withRoutes([]),
         TranslateModule.forRoot(),
       ],
+      providers: [
+        { provide: AuthService, useClass: MockAuthService }
+      ],
```
Comment (SandroGlueck) [inline src/app/main-layout/main-layout.component.spec.ts:23]: `ng lint` complains about the trailing spaces on that line.

image

File: src/app/app.module.ts (hunk 1 above)
Comment (Th3da) [reply src/app/app.module.ts:85]: No, it's not required. Have deleted it.

File: src/app/core/components/imprint/imprint.component.scss (hunk 2 above)
Comment (Th3da) [reply src/app/core/components/imprint/imprint.component.scss:29]: done

File: src/app/shared/breadcrumb/breadcrumb/breadcrumb.component.html (hunk 3 above)
Comment (Th3da) [reply src/app/shared/breadcrumb/breadcrumb/breadcrumb.component.html:8]: done

File: src/app/main-layout/main-layout.component.spec.ts (hunk 4 above)
Comment (Th3da) [reply src/app/main-layout/main-layout.component.spec.ts:23]: done.

**Review by SandroGlueck — APPROVED** [review]

**Pull request merged**